ui/app/components/meta-table/index.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="artifact-overview">
+  <table>
+    <tbody>
+      {{#if @statusReport}}
```

Comment:
Given that this component is representing data from a status report, what do you think about calling it something more explicit like `<StatusReportMetaTable>`?